skies_of_land/map.conf

```
@@ -0,0 +1,21 @@
+game_modes = return {"classic"}
+barrier_area = return {pos1={y=79,x=111,z=221},pos2={y=1,x=111,z=1}}
+teams = local _={};_[1]="flag_pos";_[2]="enabled";return {orange={pos2={y=79,x=1,z=221},[_[2]]=true,pos1={y=1,x=111,z=1},[_[1]]={y=46,x=21,z=111}},blue={pos2={y=1,x=111,z=1},[_[2]]=true,pos1={y=79,x=221,z=221},[_[1]]={y=46,x=201,z=111}}}
+chests = return {{pos2={y=40,x=155,z=79},pos1={y=38,x=143,z=67},amount=18},{pos2={y=40,x=155,z=155},pos1={y=38,x=143,z=143},amount=18},{pos2={y=40,x=79,z=155},pos1={y=38,x=67,z=143},amount=18},{pos2={y=40,x=79,z=79},pos1={y=38,x=67,z=67},amount=18},{pos2={y=49,x=209,z=209},pos1={y=47,x=201,z=201},amount=30},{pos2={y=49,x=21,z=21},pos1={y=47,x=13,z=13},amount=30}}
+phys_jump = 1
+phys_speed = 1.2
+size = return {y=80,x=222,z=222}
+map_version = 2
+enabled = true
+name = Skies of Land
+phys_gravity = 1
+initial_stuff = return {"default:cobble 10","default:torch 5","default:stick 5","ctf_ranged:pistol_loaded 1"}
+author = daveistaken
+hint = Search the sandwell outside your base for ores buried in the sand. For even more valuable ores, check beneath in the heart of the lighthouse.
+license = CC BY-SA 4.0
+others = With assistance of @Kat, @GreenBlob, @HobbitPower and @s20 (for finding bugs and giving information), A floating map designed only for Classic.
+treasures = grenades:frag_sticky;1;1;2;0.1;1;
```

```suggestion
treasures =
```
The sticky grenades are not working properly afaik.  So remove that please.

---

ok, but, after resolve this, Are the Frag sticky grenades in progress?